cypress.config.ts

```
@@ -0,0 +1,106 @@
+      // Task registration
+      on('task', {
+        log(message) {
+          console.log(message);
+          return null;
+        },
+      });
+
+      // File handling tasks
+      on('task', {
+        readFileMaybe(filename) {
+          if (require('fs').existsSync(filename)) {
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)